register.php

```diff
@@ -19,7 +19,7 @@ function addTimeTableItemToMenu(array $menuStructure): array
         'tooltip' => 'View TimeTable',
         'href' => '/TimeTable/TimeTable',
         'active' => ['TimeTable'],
-        'module' => 'tickets',
+        'module' => 'TimeTable',
     ];
```

Comment:
I'd like a comment on all of the curiosities we discovered when creating new menu items........ :)